ServerCore/Pages/Teams/Index.cshtml.cs

```diff
@@ -15,7 +15,7 @@ public IndexModel(PuzzleServerContext serverContext, UserManager<IdentityUser> u
 
         public async Task<IActionResult> OnGetAsync()
         {
-            await LoadTeamDataAsync();
+            await LoadTeamDataAsync(false);
```

Comment:
can you use a named parameter here? basically, replace "false" with "onlyIncludeTeamsLookingForMembers: false". Same for the case where you pass true.

Reply:
Removed the parameter since the check got moved to just the list html file